Framework/docs/guide/issues.md

```diff
@@ -0,0 +1,16 @@
+# Known issues
+
+This list should be updated when a rare problem occures to speed up resolution next time.
+
+###### `npm install` outputs error "Unexpected end of JSON input while parsing near ..."
+Run `npm cache clean --force`
+
+###### UI gets duplicated elements after each click
+The use of array of array of vnode might break the template engine. Avoid `h('div', [[h('span')]])`. Prefer `h('div', [h('span')])`.
+
+###### UI keeps some class names while component has been replaced by another one
+Add a key on both components so the template engine knows it's not the same anymore.
```

Comment:
What do you mean by adding a key?

Reply:
Keys for vnodes: https://github.com/AliceO2Group/WebUi/blob/dev/Framework/docs/guide/template-engine.md#keys-in-hyperscript